tests/test_views.py

```diff
@@ -0,0 +1,161 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+class TestPublicManufacturerList(TestCase):
+    def setUp(self):

```

Comment:
good job, but what about tests for POST-requests? something like creation of new object and so on